person_class.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'UNKNOWN', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public :can_use_services
```

Comment:
- [ ] There is no need specifying this `can_use_service` here. Additionally it is not the correct name. Just remove the `:can_use_services` and leave the `public`

Reply:
correct name?

Reply:
Every method under the public keyword is public so there's no need specifying the method here. Also you're missing '?' in the name there.